### PR TITLE
Update to 2.0.2

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: harbor-helm/values.yaml [hunk 1]
```diff
@@ -356,7 +356,7 @@ proxy:
 nginx:
   image:
     repository: registry.suse.de/devel/caps/registry/2.0/containers/registry/harbor-nginx
-    tag: 2.0.1-rev1
```
Comment (stefannica) [inline harbor-helm/values.yaml:359]: You should keep the `rev1` bit.

File: harbor-helm/values.yaml [hunk 2]
```diff
@@ -373,7 +373,7 @@ nginx:
 portal:
   image:
     repository: registry.suse.de/devel/caps/registry/2.0/containers/registry/harbor-portal
-    tag: 2.0.1-rev1
```
Comment (stefannica) [inline harbor-helm/values.yaml:376]: Same here: keep the `rev1` bit.

**Review by stefannica — CHANGES_REQUESTED** [review]
Overall, looks good, but you should include the revision numbers in the helm chart image tags, or at least explain why you don't think they are needed anymore.

File: harbor-helm/values.yaml (hunk 1 above)
Comment (dirkmueller) [reply harbor-helm/values.yaml:359]: done

File: harbor-helm/values.yaml (hunk 2 above)
Comment (dirkmueller) [reply harbor-helm/values.yaml:376]: done

**Review by stefannica — APPROVED** [review]

**Pull request merged**